gcloud/README.md

```diff
@@ -21,7 +21,7 @@ $ gem install gcloud
 ## Authentication
 
 Instructions and configuration options are covered in the [Authentication
-Guide](https://googleapis.github.io/google-cloud-ruby/docs/gcloud/latest/file.AUTHENTICATION).
+Guide](./AUTHENTICATION.md).
```

Comment:
This will direct users to the file on GitHub, but none of the YARD links will work on GitHub. This is why we were redirecting users to the hosted YARD output.

Reply:
Good catch. In my ideal world the github links keep you on github and the docsite links keep you on googleapis.dev. I've updated the docs publishing process, lmk what you think.

Reply:
Going to route everything to googleapis.dev